### PR TITLE
Rust & Tell May announcement 

### PR DESCRIPTION
**Pull request by evanxg852000**

### CODE REVIEW

File: _posts/2021-05-25-rust-and-tell.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  speaker:
+    name: "Senyo Simpson"
+- title: "Building a cryptocurrency derivative exchange with Rust"
+  speaker:
+    name: "Konstantin Wünscher & Blas Rodriguez"
+---
+
+We are happy 🥳 🎉 to announce that the "Rust and Tell" online meetup for May will be on Tuesday, 25th!.
+
+**SPEAKERS WANTED 🙏**
```
Comment (badboy) [inline _posts/2021-05-25-rust-and-tell.md:22]: Might want to call out that we would need speakers for after this one?
Looks like you already got 3 talks for May :)

**Review by badboy — APPROVED** [review]

File: _posts/2021-05-25-rust-and-tell.md (hunk 1 above)
Comment (evanxg852000) [reply _posts/2021-05-25-rust-and-tell.md:22]: Yes

File: _posts/2021-05-25-rust-and-tell.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+We desperatly need speakers for the following months; as we have exhausted our backlog of talks. Rust & Tell needs your help to keep going by proposing a talk. Send a message to [@evanxg852000](https://twitter.com/evanxg852000) or an email to [contact@berline.rs](mailto:contact@berline.rs).
```
Comment (badboy) [inline _posts/2021-05-25-rust-and-tell.md:24]: IMO we shouldn't be "desperate".
Something more like: "We're now looking for more speakers for the following months. Have something to talk about, a new project you're exploring or a new topic you just learned? ..."

What do you think?

Comment (evanxg852000) [reply _posts/2021-05-25-rust-and-tell.md]:
```
We're now looking for more speakers for the following months. Have something to talk about, a new project you're exploring or a new topic you just learned? The Rust & Tell community will be delighted to hear from you. Send a message to [@evanxg852000](https://twitter.com/evanxg852000) or an email to [contact@berline.rs](mailto:contact@berline.rs).
```

Comment (evanxg852000) [reply _posts/2021-05-25-rust-and-tell.md]: Just pushed a change, thanks for the quick feedback

**Review by badboy — APPROVED** [review]

**Pull request merged**